### ISSUE TITLE
touch command doesnt work as expected for multiple files

### ISSUE THREAD

**Issue by VineshReddy** [ranger/ranger#1812]
This is how it produces output with spaces instead of 2 files: 
<a href="https://imgbb.com/"><img src="https://i.ibb.co/9ndNcVq/Screenshot-from-2020-01-19-21-37-04.png" alt="Screenshot-from-2020-01-19-21-37-04" border="0"></a>
<a href="https://imgbb.com/"><img src="https://i.ibb.co/hX3FbrN/Screenshot-from-2020-01-19-21-36-29.png" alt="Screenshot-from-2020-01-19-21-36-29" border="0"></a>

**Comment by SHV007** [comment]
I think we need to make a for loop with a list.
I'm a beginner and still trying to make it work...

**Comment by toonn** [comment]
That's a very small part of the story. The problem is spaces are absolutely valid characters in a file name. So we'd have to make the `:touch` command parse shell escapes properly. On top of that some people expect `:touch` to focus the created file, that doesn't work yet but there's work in progress on it. If you can pass multiple files that can't really work, you could only focus on the first or last argument.

**Comment by SHV007** [comment]
I think what we need to do is to make touch command act like the shell command.
So let's say that we want to create multiple files, than we'll need to add comma after every file name.
like so:
`touch file1, file2`

**Comment by toonn** [comment]
No, that would create two files. One called `file1,` the other `file2`. That's not what you'd want. The *only* solution here would be to have `:touch` respect shell escaping.

**Comment by hektr** [comment]
touch as ranger command only creates one file.
touch as shell command can create multiple files, but you have to backslash special signs:
`:shell touch file\ 1 file\ 2 file\ 3`
For shell space is a sign, that divides one argument from another.
